logistics/scripts/modules/config/config-management.psm1

```diff
@@ -74,20 +74,21 @@ function Get-ExcludedExtensionSourcesFromConfig {
     [CmdletBinding()] param(
         [Parameter(Mandatory = $true)] $configFile
     )
-    $configDoc = New-Object System.Xml.XmlDocument
-    $configDoc.Load($configFile)
-    $excludedExtensionSourcesValue = ($configDoc.SelectSingleNode('/configuration/appSettings/add[@key="ExcludedExtensionSources"]/@value')).Value
 
-    if ($null -eq $excludedExtensionSourcesValue) { return "" }
-    return $excludedExtensionSourcesValue.Split(',')
+    $jsonFromFile = (Get-Content $configFile -Raw -Encoding UTF8 | ConvertFrom-JSON)
+
+    $jsonExcludedExtensions =$jsonFromFile.ApiSettings.ExcludedExtensions
+
+    if ($jsonExcludedExtensions.Length -eq 0) { return "" }
+    return $jsonExcludedExtensions
 }
 
 function Get-Configuration {
     param(
-        [string] $configFile = $folders.base.invoke('Application\EdFi.Ods.WebApi\Web.Base.config')
+        [string] $configFile = $folders.base.invoke('Application\EdFi.Ods.WebApi.NetCore\appsettings.Development.json')
```

Comment:
we should be using the production version of the appsettings.json

Reply:
Modified the code to read data from appsettings.json